### DISCUSSION TITLE
Lesson - 9 (Issue in Coverage)

### DISCUSSION THREAD

**Discussion by hard1k7724** [Cyfrin/foundry-full-course-cu#1549] [Q&A] [answered]
After running just a SINGLE test I am getting 100% coverage in my DeployRaffle.s.sol script and I think it should be very low inetead of the 100%.
Please help, I am really confused.

**Comment by king-null** [comment]
that is the only test to run in DeployRaffle at the moment that is why is 100%

**Comment by EngrPips** [comment, marked as answer]
Hello @Juice-07 , I would assume foundry is accurate in whatever coverage it calculates and keep moving forward with the course.

**Reply by hard1k7724** [reply]
Hey @EngrPips, Thanks a lot.

**Reply by EngrPips** [reply]
Anytime friend, i am glad i could help.
